Servers/UI/OJS.Servers.Ui/ClientApp/src/pages/home/HomeInfoPage.tsx

```diff
@@ -0,0 +1,119 @@
+import React, { useCallback } from 'react';
+import { IconType } from 'react-icons';
+import { FaCode, FaDeezer, FaPuzzlePiece, FaTasks, FaTrophy, FaUsers } from 'react-icons/fa';
+import LiteYouTubeEmbed from 'react-lite-youtube-embed';
+
+import IconSize from '../../components/guidelines/icons/common/icon-sizes';
+import Icon from '../../components/guidelines/icons/Icon';
+import SpinningLoader from '../../components/guidelines/spinning-loader/SpinningLoader';
+import useTheme from '../../hooks/use-theme';
+import { useGetHomeStatisticsQuery } from '../../redux/services/homeStatisticsService';
+import { flexCenterObjectStyles } from '../../utils/object-utils';
+import withTitle from '../shared/with-title';
+
+import 'react-lite-youtube-embed/dist/LiteYouTubeEmbed.css';
+import styles from './HomeInfoPage.module.scss';
+
+interface IHomePageStatistic {
+    title: string;
+    iconType: IconType;
+    count: string;
+}
+
+const HomePageStatistic = (props: IHomePageStatistic) => {
+    const { title, iconType, count } = props;
+
+    const { themeColors, getColorClassName } = useTheme();
+    const textColorClassName = getColorClassName(themeColors.textColor);
+
+    return (
+        <div className={styles.homePageStatisticWrapper}>
+            <Icon
+              size={IconSize.ExtraLarge}
+              Component={iconType}
+              color={themeColors.baseColor100}
+            />
+            <div className={styles.homePageStatisticTextWrapper}>
+                <div style={{ color: '#44a9f8' }}>{title}</div>
```

Comment:
Is this color compatible with the light theme?

Reply:
This is just moved file, I didn't introduce this color now..